docs/packages/only_wiki/glkit.rst

```diff
@@ -0,0 +1,24 @@
+.. spelling::
+
+    glkit
+
+.. _pkg.glkit:
+
+glkit
+=====
+
+File only available in wiki.
+Please merge with hunter-package entry
+
+.. code-block:: cmake
+
+    find_package(glkit REQUIRED)
+    target_link_libraries(... glkit::glkit)
```

Comment:
I think pattern `target_link_libraries(...` can be substituted with `target_link_libraries(foo` so we don't have warning about incorrect syntax.

Reply:
the incorrect syntax was because of the line only containing `...`. the other lines compile fine (the current PR compiles the html fine, but has spelling errors